schema/gen/go/testEngine_test.go

```diff
@@ -30,23 +33,25 @@ func genAndCompileAndTest(
 	testsFn behavioralTests,
 ) {
 	// Make directories for the package we're about to generate.
-	//  Everything will be prefixed with "./_test".
-	// You can rm-rf the whole "./_test" dir at your leisure.
-	//  We don't by default because it's nicer to let go's builds of things cache.
-	//  If you change the names of types, though, you'll have garbage files leftover,
-	//   and that's currently a manual cleanup problem.  Sorry.
-	os.Mkdir("./_test/", 0755)
-	os.Mkdir("./_test/"+prefix, 0755)
+	// They will live in a temporary directory, usually
+	// /tmp/go-ipld-prime-gengo on Linux. It can be removed at any time.
+	// We don't by default because it's nicer to let go's builds of things cache.
+	// If you change the names of types, though, you'll have garbage files leftover,
+	// and that's currently a manual cleanup problem.  Sorry.
```

Comment:
Oh, heh.  Not exactly related to your change, but since it shows up in the diff: we could remove these two lines now.  We no longer have this problem :D